collector/logind_linux.go

```diff
@@ -0,0 +1,241 @@
+// Copyright 2016 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !nologind
+
+package collector
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/godbus/dbus"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	logindSubsystem = "logind"
+	dbusObject      = "org.freedesktop.login1"
+	dbusPath        = "/org/freedesktop/login1"
+)
+
+var (
+	// Taken from logind as of systemd v229
```

Comment:
I see. I guess I should add an "other" value to the type and class attributes to catch these.